### PR TITLE
Update TLDs to version 2024032200

### PR DESCRIPTION
**Pull request by rossanthony**

### CODE REVIEW

**Comment by rossanthony** [comment]
@Marsup tagging you in case you hadn't seen this

**Review by Nargonath — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "@hapi/tlds",
     "description": "TLDS list for domain validation",
-    "version": "1.0.2",
+    "version": "1.0.3",
```
Comment (Marsup) [inline package.json]: Never bump version when submitting a PR
```suggestion
    "version": "1.0.2",
```

**Review by Marsup — COMMENTED** [review]
Thanks, I'll release it shortly.

File: package.json (hunk 1 above)
Comment (Nargonath) [reply package.json]: Oh yeah right, I missed it.

**Pull request merged**